# HG changeset patch
# User Carnë Draug <carandraug@octave.org>
# Date 1383437445 0
#      Sun Nov 03 00:10:45 2013 +0000
# Node ID efbed19b8e0478bdfea389248efad7e3950b54d4
# Parent  7957aedf0b2d998a06bd5733ac7817042f9f53a5
Add Portuguese (Portugal) translation.

* libgui/languages/pt_PT.ts: new Octave Portuguese (Portugal) translation
based on pt_BR.
* libgui/languages/translators: Add Carnë Draug to translator and contact
for pt_PT.
* lingui/Makefile.am: add pt_PT translation to build system.

diff --git a/libgui/Makefile.am b/libgui/Makefile.am
--- a/libgui/Makefile.am
+++ b/libgui/Makefile.am
@@ -27,16 +27,17 @@ octlib_LTLIBRARIES = liboctgui.la
 TRANSLATIONS = \
   languages/be_BY.ts \
   languages/de_DE.ts \
   languages/en_US.ts \
   languages/es_ES.ts \
   languages/fr_FR.ts \
   languages/nl_NL.ts \
   languages/pt_BR.ts \
+  languages/pt_PT.ts \
   languages/ru_RU.ts \
   languages/uk_UA.ts
 
 LOCALES = $(patsubst languages/%.ts, languages/%.qm, $(TRANSLATIONS))
 
 EXTRA_DIST = default-qt-settings.in $(TRANSLATIONS)
 CLEANFILES =
 
diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_PT.ts
copy from libgui/languages/pt_BR.ts
copy to libgui/languages/pt_PT.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_PT.ts
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="pt_BR" sourcelanguage="en">
+<TS version="2.0" language="pt_PT" sourcelanguage="pt_BR">
 <context>
     <name>ListDialog</name>
     <message>
         <location filename="../src/dialog.cc" line="+250"/>
         <source>Select All</source>
-        <translation>Selecionar Tudo</translation>
+        <translation>Seleccionar Tudo</translation>
     </message>
 </context>
 <context>
     <name>QObject</name>
     <message>
         <location filename="../src/workspace-model.cc" line="+75"/>
         <source>automatic</source>
         <translation>automático(a)</translation>
@@ -49,17 +49,17 @@
     <message>
         <location line="+1"/>
         <source>background</source>
         <translation>fundo</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>selection</source>
-        <translation>seleção</translation>
+        <translation>selecçao</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
         <translation>indicador</translation>
     </message>
 </context>
 <context>
@@ -80,17 +80,17 @@
         <translation>Limpar Tudo</translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1527"/>
         <source>copied selection to clipboard</source>
-        <translation>seleção copiada para a área de transferência</translation>
+        <translation>selecçao copiada para a área de transferência</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Documentação</translation>
@@ -110,54 +110,56 @@
         <location line="+57"/>
         <location line="+28"/>
         <source>Octave Editor</source>
         <translation>Editor do Octave</translation>
     </message>
     <message>
         <location line="-249"/>
         <source>Octave Files (*.m);;All Files (*)</source>
-        <translation>Scripts do Octave (*.m);;Todos os Arquivos (*)</translation>
+        <translation>Scripts de Octave (*.m);;Todos os Ficheiros (*)</translation>
     </message>
     <message>
         <location line="+222"/>
         <source>File not saved! A file with the selected name
 %1
 is already open in the editor</source>
-        <translation>Arquivo não salvo! Um arquivo com o nome selecionado %1 já está aberto no editor</translation>
+        <translation>Ficheiro não foi gravado! Um ficheiro com o nome selecionado
+%1
+já está aberto no editor</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>The associated file editor tab has disappeared.  It was likely closed by some means.</source>
-        <translation>A aba do editor associada ao arquivo desapareceu. Provavelmente foi fechada de alguma maneira.</translation>
+        <translation>A aba do editor associada ao ficheiro desapareceu. Foi provavelmente fechada de alguma maneira.</translation>
     </message>
     <message>
         <location line="+228"/>
         <source>&amp;%1 %2</source>
         <translation>&amp;%1 %2</translation>
     </message>
     <message>
         <location line="+173"/>
         <source>&amp;New File</source>
-        <translation>&amp;Novo Arquivo</translation>
+        <translation>&amp;Novo Ficheiro</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Open File</source>
-        <translation>&amp;Abrir Arquivo</translation>
+        <translation>&amp;Abrir Ficheiro</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Save File</source>
-        <translation>&amp;Salvar Arquivo</translation>
+        <translation>&amp;Guardar Ficheiro</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save File &amp;As</source>
-        <translation>Salvar Arquivo &amp;Como</translation>
+        <translation>Guardar Ficheiro &amp;Como</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Print</source>
         <translation>Imprimir</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -172,17 +174,17 @@ is already open in the editor</source>
     <message>
         <location line="+3"/>
         <source>&amp;Copy</source>
         <translation>&amp;Copiar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Cu&amp;t</source>
-        <translation>Recor&amp;tar</translation>
+        <translation>Cor&amp;tar</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Paste</source>
         <translation>Colar</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -217,32 +219,32 @@ is already open in the editor</source>
     <message>
         <location line="+3"/>
         <source>Toggle &amp;breakpoint</source>
         <translation>Habilitar/desabilitar ponto de &amp;interrupção</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Remove All breakpoints</source>
-        <translation>&amp;Remover Todos Pontos de interrupção</translation>
+        <translation>&amp;Remover Todos os Pontos de interrupção</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Comment</source>
         <translation>&amp;Comentar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Uncomment</source>
         <translation>&amp;Descomentar</translation>
     </message>
     <message>
         <location line="+63"/>
         <source>&amp;Recent Editor Files</source>
-        <translation>Arquivos &amp;Recentes</translation>
+        <translation>Ficheiros &amp;recentes</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>&amp;Edit Function</source>
         <translation>&amp;Editar Função</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -252,84 +254,84 @@ is already open in the editor</source>
     <message>
         <location line="+4"/>
         <source>Close All</source>
         <translation>Fechar Todos</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Close Other Files</source>
-        <translation>Fechar Outros Arquivos</translation>
+        <translation>Fechar os Outros Ficheiros</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Run &amp;Selection</source>
-        <translation>Executar &amp;Seleção</translation>
+        <translation>Executar &amp;Selecção</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
         <translation>&amp;Ajuda</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Help on Keyword</source>
-        <translation>&amp;Ajuda na Palavra-chave</translation>
+        <translation>&amp;Ajuda da Palavra-chave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Documentation on Keyword</source>
-        <translation>&amp;Documentação na Palavra-chave</translation>
+        <translation>&amp;Documentação da Palavra-chave</translation>
     </message>
     <message>
         <location line="-147"/>
         <source>&amp;Find and Replace</source>
         <translation>&amp;Buscar e Substituir</translation>
     </message>
     <message>
         <location line="-584"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation>Não foi possível abrir aquivo
+        <translation>Não foi possível abrir o ficheiro
 %1
 para leitura: %2.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
-        <translation>Arquivo
+        <translation>O ficheiro
 %1
 não existe. Deseja criá-lo?</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Could not open file
 %1
 for write: %2.</source>
-        <translation>Não foi possível abrir arquivo
+        <translation>Não foi possível abrir o ficheiro
 %1
 para escrita: %2.</translation>
     </message>
     <message>
         <location line="+549"/>
         <source>Save File And Run</source>
-        <translation>Salvar Arquivo e Executar</translation>
+        <translation>Guardar Ficheiro e Executar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Go&amp;to Line</source>
-        <translation>Vá para &amp;Linha</translation>
+        <translation>Ir para &amp;Linha</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>&amp;File</source>
-        <translation>&amp;Arquivo</translation>
+        <translation>&amp;Ficheiro</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>&amp;Edit</source>
         <translation>&amp;Editar</translation>
     </message>
     <message>
         <location line="+22"/>
@@ -352,312 +354,316 @@ para escrita: %2.</translation>
         <translation>&amp;Executar</translation>
     </message>
 </context>
 <context>
     <name>file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+755"/>
         <source>Goto line</source>
-        <translation>Vá para linha</translation>
+        <translation>Ir para linha</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Line number</source>
         <translation>Número da linha</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>&lt;unnamed&gt;</source>
         <translation>&lt;sem_nome&gt;</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Do you want to save or discard the changes?</source>
-        <translation>Você deseja salvar ou descartar as alterações?</translation>
+        <translation>Você deseja guardar ou descartar as alterações?</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Do you want to cancel closing, save or discard the changes?</source>
-        <translation>Você deseja cancelar fechamento, salvar ou descartar as alterações?</translation>
+        <translation>Você deseja cancelar o fecho, guardar, ou descartar as alterações?</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+129"/>
         <location line="+115"/>
         <location line="+66"/>
         <location line="+22"/>
         <source>Octave Editor</source>
         <translation>Editor do Octave</translation>
     </message>
     <message>
         <location line="-331"/>
         <source>The file
 %1
 is about to be closed but has been modified.
 %2</source>
-        <translation>O arquivo
+        <translation>O ficheiro
 %1
 está prestes a ser fechado mas foi modificado.
 %2</translation>
     </message>
     <message>
         <location line="+210"/>
         <source>Octave Files (*.m);;All Files (*)</source>
-        <translation>Scripts Octave (*.m);; Todos os Arquivos (*)</translation>
+        <translation>Ficheiros de Octave (*.m);; Todos os Ficheiros (*)</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>File not saved! The selected file name
 %1
 is the same as the current file name</source>
-        <translation>Arquivo não salvo! O nome selecionado
+        <translation>O ficheiro não foi guardado! O nome seleccionado
 %1
-é o mesmo do arquivo atual</translation>
+é o mesmo do que o ficheiro actual</translation>
     </message>
     <message>
         <location line="+81"/>
         <source>
 
 Warning: The contents in the editor is modified!</source>
         <translation>
 
 Aviso: O conteúdo no editor foi modificado!</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>It seems that the file
 %1
 has been deleted or renamed. Do you want to save it now?%2</source>
-        <translation>Parece que o arquivo
+        <translation>Parece que o ficheiro
 %1
-foi removido ou renomeado. Pretende salvá-lo agora?%2</translation>
+foi removido ou renomeado. Pretende guardá-lo agora?%2</translation>
     </message>
     <message>
         <location line="-203"/>
         <source>Could not open file %1 for write:
 %2.</source>
-        <translation>Não foi possível abrir arquivo %1 para escrita:
+        <translation>Não foi possível abrir o ficheiro %1 para escrita:
 %2.</translation>
     </message>
     <message>
         <location line="+181"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
-        <translation>Parece que o arquivo &apos;%1&apos; foi modificado por outra aplicação. Deseja recarregá-lo?</translation>
+        <translation>Parece que o ficheiro &apos;%1&apos; foi modificado por outro aplicativo. Deseja recarregá-lo?</translation>
     </message>
 </context>
 <context>
     <name>files_dock_widget</name>
     <message>
         <location filename="../src/files-dock-widget.cc" line="+67"/>
         <source>File Browser</source>
-        <translation>Navegador de Arquivos</translation>
+        <translation>Navegador de Ficheiros</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse your files.</source>
-        <translation>Explore seus arquivos.</translation>
+        <translation>Navegue pelos seus ficheiros.</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Enter the path or filename</source>
-        <translation>Digite o caminho ou nome de arquivo</translation>
+        <translation>Introduza o caminho ou nome do ficheiro</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Move up one directory</source>
-        <translation>Subir um nível de diretório</translation>
+        <translation>Subir um nível de directório</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show octave directory</source>
-        <translation>Exibir o diretório do Octave</translation>
+        <translation>Mostrar o directório actual do Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Goto current octave directory</source>
-        <translation>Vá para o diretório atual do Octave</translation>
+        <translation>Ir para o directório actual do Octave</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Set octave directory</source>
-        <translation>Definir o diretório do Octave</translation>
+        <translation>Definir o directório actual do Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Set octave directroy to current browser directory</source>
-        <translation>Definir diretório do Octave igual ao diretório atual no navegador</translation>
+        <translation>Definir o directório actual do Octave para o directório actual do navegador</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Actions on current directory</source>
-        <translation>Ações no diretório atual</translation>
+        <translation>Acções no directório actual</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Home directory</source>
-        <translation>Exibir diretório pessoal</translation>
+        <translation>Mostrar directório pessoal</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Search directory</source>
-        <translation>Buscar no diretório</translation>
+        <translation>Procurar no directório</translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+244"/>
         <source>Find Files ...</source>
-        <translation>Buscar arquivos...</translation>
+        <translation>Procurar Ficheiros...</translation>
     </message>
     <message>
         <location line="-240"/>
         <location line="+252"/>
         <source>New File</source>
-        <translation>Novo Arquivo</translation>
+        <translation>Novo Ficheiro</translation>
     </message>
     <message>
         <location line="-249"/>
         <location line="+252"/>
         <source>New Directory</source>
-        <translation>Novo Diretório</translation>
+        <translation>Novo Directório</translation>
     </message>
     <message>
         <location line="-223"/>
         <source>Doubleclick a file to open it</source>
-        <translation>Clique duas vezes em um arquivo para abrí-lo</translation>
+        <translation>Clique duas vezes num ficheiro para abri-lo</translation>
     </message>
     <message>
         <location line="+185"/>
         <source>Open</source>
         <translation>Abrir</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Open in Default Application</source>
-        <translation>Abrir na Aplicação Padrão</translation>
+        <translation>Abrir no Aplicativo Padrão</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Copy Selection to Clipboard</source>
-        <translation>Copiar Seleção para Área de Transferência</translation>
+        <translation>Copiar Selecção para Área de Transferência</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Run</source>
         <translation>Executar</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Load Data</source>
         <translation>Carregar Dados</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set Current Directory</source>
-        <translation>Definir Diretório Atual</translation>
+        <translation>Definir Directório Actual</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Rename</source>
         <translation>Renomear</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Delete</source>
         <translation>Excluir</translation>
     </message>
     <message>
         <location line="+107"/>
         <source>Rename file/directory</source>
-        <translation>Renomear arquivo/diretório</translation>
+        <translation>Renomear ficheiro/directório</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Rename file/directory:
 </source>
-        <translation>Renomear arquivo/diretório:</translation>
+        <translation>Renomear ficheiro/directório:
+</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>
  to: </source>
         <translation>
-para:</translation>
+ para:</translation>
     </message>
     <message>
         <location line="+25"/>
         <location line="+11"/>
         <source>Delete file/directory</source>
-        <translation>Remover arquivo/diretório</translation>
+        <translation>Remover ficheiro/directório</translation>
     </message>
     <message>
         <location line="-10"/>
         <source>Are you sure you want to delete
 </source>
-        <translation>Você tem certeza que deseja excluir</translation>
+        <translation>Tem a certeza que deseja remover
+</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
-        <translation>Não é possível excluir um diretório que não está vázio</translation>
+        <translation>Não é possível remover um directório que não está vazio</translation>
     </message>
     <message>
         <location line="+131"/>
         <source>Set directory of file browser</source>
-        <translation>Definir diretório do navegador de arquivos</translation>
+        <translation>Definir directório do navegador de ficheiros</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Create File</source>
-        <translation>Criar Arquivo</translation>
+        <translation>Criar Ficheiro</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Create file in
 </source>
-        <translation>Criar arquivo em</translation>
+        <translation>Criar ficheiro em
+</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Create Directory</source>
-        <translation>Criar Diretório</translation>
+        <translation>Criar Directório</translation>
     </message>
     <message>
         <location line="+0"/>
         <source>Create folder in
 </source>
-        <translation>Criar diretório em</translation>
+        <translation>Criar directório em
+</translation>
     </message>
 </context>
 <context>
     <name>find_dialog</name>
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+77"/>
         <source>Find &amp;what:</source>
-        <translation>&amp;Buscar por:</translation>
+        <translation>Procurar por:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Re&amp;place with:</source>
         <translation>&amp;Substituir por:</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Match &amp;case</source>
-        <translation>Considerar &amp;maiúsculas/minúsculas</translation>
+        <translation>Equivaler capitalização</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search from &amp;start</source>
-        <translation>Buscar a paritr do &amp;início</translation>
+        <translation>Procurar a partir do início</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Wrap while searching</source>
-        <translation>Retornar ao começo se encontrar &amp;final durante busca</translation>
+        <translation>Dar a volta durante a procura</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Find Next</source>
         <translation>Buscar &amp;Próximo</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -677,42 +683,42 @@ para:</translation>
     <message>
         <location line="+2"/>
         <source>&amp;More</source>
         <translation>&amp;Mais</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>&amp;Whole words</source>
-        <translation>Palavras &amp;inteiras</translation>
+        <translation>Palavras inteiras</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Regular E&amp;xpressions</source>
         <translation>Expressões &amp;Regulares</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search &amp;backward</source>
-        <translation>Buscar para &amp;trás</translation>
+        <translation>Procurar para trás</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search se&amp;lection</source>
-        <translation>Buscar dentre &amp;seleção</translation>
+        <translation>Procurar dentro da selecção</translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Search from end</source>
-        <translation>Buscar desde o final</translation>
+        <translation>Procurar a partir do fim</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Search from start</source>
-        <translation>Buscar desde o início</translation>
+        <translation>Procurar a partir do início</translation>
     </message>
     <message>
         <location line="+117"/>
         <source>Replace Result</source>
         <translation>Substituir Resultado</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -722,191 +728,191 @@ para:</translation>
     <message>
         <location line="+10"/>
         <source>Find Result</source>
         <translation>Buscar Resultado</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>No more matches found</source>
-        <translation>Nenhum resultado encontrado</translation>
+        <translation>Não foi encontrado mais nenhum resultado</translation>
     </message>
 </context>
 <context>
     <name>find_files_dialog</name>
     <message>
         <location filename="../src/find-files-dialog.cc" line="+47"/>
         <source>Find Files</source>
-        <translation>Buscar Arquivos</translation>
+        <translation>Procurar Ficheiros</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Named:</source>
-        <translation>Nomeados:</translation>
+        <translation>Chamados:</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Enter the filename expression</source>
-        <translation>Digite expressão para os nomes de arquivo</translation>
+        <translation>Introduza a expressão para o nome do ficheiro</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Start in:</source>
         <translation>Iniciar em:</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Enter the start directory</source>
-        <translation>Digite diretório inicial</translation>
+        <translation>Introduzir o directório inicial</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse...</source>
-        <translation>Explorar...</translation>
+        <translation>Navegar...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse for start directory</source>
-        <translation>Explorar diretório inicial</translation>
+        <translation>Navegar por directório inicial</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Recurse directories</source>
-        <translation>Entrar em sub-diretórios</translation>
+        <translation>Entrar em sub-directórios</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Search recursively through directories for matching files</source>
-        <translation>Buscar recursivamente nos diretórios por arquivos que correspondam</translation>
+        <translation>Procurar recursivamente pelos directórios por ficheiros correspondentes</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Include directories</source>
-        <translation>Inclur diretórios</translation>
+        <translation>Incluir directórios</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Include matching directories in search results</source>
-        <translation>Incluir diretórios encontrados no resultado da busca</translation>
+        <translation>Incluir directórios correspondentes nos resultado da pesquisa</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Name case insensitive</source>
-        <translation>Insensível a maiúsculas/minúsculas</translation>
+        <translation>Insensível à capitalização</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Set matching name is case insensitive</source>
-        <translation>Definir se nome é insensível a maúscula/minúscula</translation>
+        <translation>Definir se o nome é insensível à capitalização</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Contains text:</source>
         <translation>Contém texto:</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Search must match text</source>
-        <translation>Busca deve encontrar texto</translation>
+        <translation>Pesquisa precisa corresponder ao texto</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Text to match</source>
-        <translation>Texto para encontrar</translation>
+        <translation>Texto para corresponder</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Text case insensitive</source>
-        <translation>Texto insensível a maiúscula/minúscula</translation>
+        <translation>Texto insensível à capitalização</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Set text content is case insensitive</source>
-        <translation>Definir se texto é insensível a maiúscula/minúscula</translation>
+        <translation>Definir se o texto é insensível à capitalização</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Search results</source>
-        <translation>Resultados da Busca</translation>
+        <translation>Resultados de pesquisa</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Idle.</source>
         <translation>Parado.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Find</source>
         <translation>Buscar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Start search for matching files</source>
-        <translation>Iniciar busca por arquivos</translation>
+        <translation>Iniciar procura por ficheiros correspondentes</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Stop</source>
         <translation>Parar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Stop searching</source>
-        <translation>Parar busca</translation>
+        <translation>Parar procura</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>File name/location</source>
-        <translation>Nome/local do arquivo</translation>
+        <translation>Nome/localização do ficheiro</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>File contents</source>
-        <translation>Conteúdo do arquivo</translation>
+        <translation>Conteúdo do ficheiro</translation>
     </message>
     <message>
         <location line="+99"/>
         <source>Searching...</source>
-        <translation>Buscando...</translation>
+        <translation>Procurando...</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Set search directory</source>
-        <translation>Alterar diretório de busca</translation>
+        <translation>Definir directório de procura</translation>
     </message>
 </context>
 <context>
     <name>find_files_model</name>
     <message>
         <location filename="../src/find-files-model.cc" line="+76"/>
         <source>Filename</source>
-        <translation>Nome do Arquivo</translation>
+        <translation>Nome do Ficheiro</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Directory</source>
-        <translation>Diretório</translation>
+        <translation>Directório</translation>
     </message>
 </context>
 <context>
     <name>history_dock_widget</name>
     <message>
         <location filename="../src/history-dock-widget.cc" line="+43"/>
         <source>Browse and search the command history.</source>
-        <translation>Naveguar e buscar no histórico de comandos.</translation>
+        <translation>Navegar e procurar no histórico de comandos.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Doubleclick a command to transfer it to the terminal.</source>
-        <translation>Clique duas vezes em um comando para transferí-lo ao terminal.</translation>
+        <translation>Clique duas vezes num comando para transferí-lo para o terminal.</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Enter text to filter the command history.</source>
-        <translation>Digite para filtrar o hitórico de comandos.</translation>
+        <translation>Introduza texto para filtrar o histórico de comandos.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Command History</source>
         <translation>Histórico de Comandos</translation>
     </message>
     <message>
         <location line="+20"/>
@@ -935,17 +941,17 @@ para:</translation>
         <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation>Sobre o Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
-        <translation>&amp;Arquivo</translation>
+        <translation>Ficheiro</translation>
     </message>
     <message>
         <location line="+54"/>
         <source>New</source>
         <translation>Novo</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -996,47 +1002,47 @@ para:</translation>
         <location line="+6"/>
         <source>Paste</source>
         <translation>Colar</translation>
     </message>
     <message>
         <location line="-1026"/>
         <location line="+946"/>
         <source>Save Workspace As</source>
-        <translation>Salvar Ambiente de Trabalho Como</translation>
+        <translation>Guardar Ambiente de Trabalho como</translation>
     </message>
     <message>
         <location line="-826"/>
         <source>The release notes file &apos;%1&apos; is empty.</source>
-        <translation>O arquivo de notas da versão &apos;%1&apos; está vazio.</translation>
+        <translation>O ficheiro de notas da versão &apos;%1&apos; está vazio.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>The release notes file &apos;%1&apos; cannot be read.</source>
-        <translation>O arquivo de notas da versão &apos;%1&apos; não pode ser lido.</translation>
+        <translation>O ficheiro de notas da versão &apos;%1&apos; não pode ser lido.</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Notas da Versão do Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
-        <translation>Definir diretório de trabalho</translation>
+        <translation>Definir directório de trabalho</translation>
     </message>
     <message>
         <location line="+736"/>
         <source>Clear Clipboard</source>
         <translation>Limpar Área de Transferência</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Find Files...</source>
-        <translation>Buscar Arquivos...</translation>
+        <translation>Encontrar Ficheiros...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Clear Command Window</source>
         <translation>Limpar Janela de Comandos</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1071,32 +1077,32 @@ para:</translation>
     <message>
         <location line="+4"/>
         <source>Continue</source>
         <translation>Continuar</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Exit Debug Mode</source>
-        <translation>Sair de Modo de Depuração</translation>
+        <translation>Sair do Modo de Depuração</translation>
     </message>
     <message>
         <location line="+48"/>
         <source>Show File Browser</source>
-        <translation>Exibir Navegador de Arquivos</translation>
+        <translation>Mostrar Navegador de Ficheiros</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Show News Window</source>
         <translation>Exibir Janela de Novidades</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>File Browser</source>
-        <translation>Navegador de Arquivos</translation>
+        <translation>Navegador de Ficheiros</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>News</source>
         <translation>Novidades</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1106,27 +1112,27 @@ para:</translation>
     <message>
         <location line="+86"/>
         <source>Octave Packages</source>
         <translation>Pacotes do Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Share Code</source>
-        <translation>Compartilhar o Código</translation>
+        <translation>Partilhar Código</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Contribute to Octave</source>
-        <translation>Contribuir com o Octave</translation>
+        <translation>Contribuir para o Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Developer Resources</source>
-        <translation>Recursos para Desenvolvedores do Octave</translation>
+        <translation>Recursos para Programadores do Octave</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>On Disk</source>
         <translation>No Disco</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1146,32 +1152,32 @@ para:</translation>
     <message>
         <location line="+3"/>
         <source>Community News</source>
         <translation>Novidades da Comunidade</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Enter directory name</source>
-        <translation>Digite nome do diretório</translation>
+        <translation>Introduzir nome do directório</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Current Directory: </source>
-        <translation>Diretório Atual:</translation>
+        <translation>Directório Actual</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>One directory up</source>
-        <translation>Um nível de diretório acima</translation>
+        <translation>Um nível de directório acima</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse directories</source>
-        <translation>Navegar por diretórios</translation>
+        <translation>Navegar directórios</translation>
     </message>
     <message>
         <location line="-445"/>
         <source>Load workspace</source>
         <translation>Carregar ambiente de trabalho</translation>
     </message>
     <message>
         <location line="+200"/>
@@ -1282,45 +1288,45 @@ para:</translation>
     <message>
         <location line="+2"/>
         <source>Edit</source>
         <translation>Editar</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Run Selection</source>
-        <translation>&amp;Executar Seleção</translation>
+        <translation>&amp;Executar Selecção</translation>
     </message>
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+270"/>
         <source>The file %1 does not exist in the load path.  To debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation>O arquivo %1 não existe no caminho de pesquisa.  Para depurar a função que você está editando, é preciso ou mudar para o diretório %2 ou adicionar esse diretório ao caminho de pesquisa.</translation>
+        <translation>O ficheiro %1 não existe no caminho de pesquisa. Para depurar a função que está a ser editada, é preciso mudar para o directório %2 ou adicionar esse directório ao caminho de pesquisa.</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path.  To debug the function you are editing, change to the directory %2.</source>
-        <translation>Existe um arquivo com o mesmo nome %1 no caminho de pesquisa. Para depurar a função que você está editando, é preciso mudar para o diretório %2.</translation>
+        <translation>O ficheiro %1 é ignorado a favor de outro ficheiro no caminho de pesquisa com o mesmo nome. Para depurar a função, é preciso mudar para o directório %2.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
-        <translation>Mudar para Diretório ou Adicionar Diretório ao Caminho de Pesquisa</translation>
+        <translation>Alterar directório ou adicionar ao caminho de pesquisa</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory</source>
-        <translation>Mudar para Diretório</translation>
+        <translation>Alterar Directório</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Add Directory to Load Path</source>
-        <translation>Adicionar diretório ao caminho de pesquisa</translation>
+        <translation>Adicionar directório ao caminho de pesquisa</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Cancel</source>
         <translation>Cancelar</translation>
     </message>
 </context>
 <context>
@@ -1338,107 +1344,107 @@ para:</translation>
     <message>
         <location line="+97"/>
         <source>Octave logo only</source>
         <translation>Apenas logotipo do Octave</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
-        <translation>Ícones de texto</translation>
+        <translation>Ícones de letras</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Graphic icons</source>
         <translation>Ícones gráficos</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>Show white space</source>
-        <translation>Exibir espaço em branco</translation>
+        <translation>Mostrar espaços em branco</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Do not show white spaces used for indentation</source>
-        <translation>Não exibir espaços em branco usados para identação</translation>
+        <translation>Não mostrar espaços em branco usados para identação</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Color</source>
         <translation>Cor</translation>
     </message>
     <message>
         <location line="+120"/>
         <source>Indent width</source>
-        <translation>Largura de identação</translation>
+        <translation>Largura da identação</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Tab indents line</source>
-        <translation>Tecla Tab causa identação na linha</translation>
+        <translation>Tabulador identa linha</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Auto indentation</source>
         <translation>Auto-identação</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Tab width</source>
-        <translation>Largura do Tab</translation>
+        <translation>Largura do tabulador</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show indentation guides</source>
         <translation>Exibir guias de identação</translation>
     </message>
     <message>
         <location line="+17"/>
         <source>Backspace unindents line</source>
         <translation>Tecla Apagar remove identação na linha</translation>
     </message>
     <message>
         <location line="+84"/>
         <source>Characters before list with suggestions is displayed</source>
-        <translation>Mínimo de caracteres para exibir lista de sugestões</translation>
+        <translation>Número de caracteres para activar lista de sugestões</translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Match keywords</source>
         <translation>Buscar palavras-chave</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Case sensitive</source>
-        <translation>Sensível a maiúscula/minúscula</translation>
+        <translation>Sensível à capitalização</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Replace word by suggested one</source>
-        <translation>Substituir palavra por aquela sugerida</translation>
+        <translation>Substituir palavra pela sugerida</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Match words in document</source>
         <translation>Buscar palavras no documento</translation>
     </message>
     <message>
         <location line="+61"/>
         <source>Restore editor tabs from previous session on startup</source>
         <translation>Restaurar abas de edição da sessão anterior durante inicialização</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Create non existing files without prompting</source>
-        <translation>Criar arquivos inexistentes sem perguntar</translation>
+        <translation>Criar ficheiros inexistentes sem perguntar</translation>
     </message>
     <message>
         <location line="+47"/>
         <source>Use custom file editor</source>
         <translation>Usar editor de texto externo</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1448,42 +1454,42 @@ para:</translation>
     <message>
         <location line="+22"/>
         <source>Editor Styles</source>
         <translation>Estilos do Editor</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as difference to the default size), font decoration (bold, italic, underline), textcolor and background color (for the latter, the color pink (255,0,255) is a placeholder for the default background color)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecionar fonte, tamanho de fonte (na forma de diferença com relação a tamanho padrão), decoração da fonte (negrito, itálico, sublinhado), cor do texto e cor de fundo de tela (para o último, a cor rosa (255,0,255) é sinônimo da cor de fundo padrão)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Seleccionar tipo de letra, tamanho (como diferença para o tamanho padrão), decoração (negrito, itálico, sublinhado), cor do texto e cor do fundo de tela (para o último, a cor rosa (255,0,255) é sinônimo da cor de fundo padrão)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+76"/>
         <source>Use Foreground Color</source>
         <translation>Usar Cor do Primeiro Plano</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Terminal Colors</source>
         <translation>Cores do Terminal</translation>
     </message>
     <message>
         <location line="+45"/>
         <source>Font</source>
-        <translation>Fonte</translation>
+        <translation>Tipo de Letra</translation>
     </message>
     <message>
         <location line="-757"/>
         <source>Show line numbers</source>
         <translation>Exibir número de linha</translation>
     </message>
     <message>
         <location line="+27"/>
         <source>Highlight current line</source>
-        <translation>Enfatizar linha atual</translation>
+        <translation>Enfatizar linha actual</translation>
     </message>
     <message>
         <location line="+262"/>
         <source>Code completion</source>
         <translation>Sguestões para copletar código</translation>
     </message>
     <message>
         <location line="-282"/>
@@ -1498,57 +1504,57 @@ para:</translation>
     <message>
         <location line="+67"/>
         <source>Terminal</source>
         <translation>Terminal</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Cursor type:</source>
-        <translation>Tipo do Indicador:</translation>
+        <translation>Tipo de Cursor:</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Cursor blinking</source>
         <translation>Indicador piscante</translation>
     </message>
     <message>
         <location line="+101"/>
         <source>Font size</source>
-        <translation>Tamanho de fonte</translation>
+        <translation>Tamanho da letra</translation>
     </message>
     <message>
         <location line="+35"/>
         <source>File Browser</source>
-        <translation>Navegador de Arquivos</translation>
+        <translation>Navegador de Ficheiros</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Show file size</source>
-        <translation>Exibir tamanho do arquivo</translation>
+        <translation>Mostrar tamanho do ficheiro</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show file type</source>
-        <translation>Exibir tipo do arquivo</translation>
+        <translation>Mostrar tipo de ficheiro</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show date of last modification</source>
-        <translation>Exibir data de última modificação</translation>
+        <translation>Mostrar data da última modificação</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show hidden files</source>
-        <translation>Exibir arquivos ocultos</translation>
+        <translation>Mostrar ficheiros escondidos</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Synchronize octave directory with the file browser</source>
-        <translation>Sincronizar diretório do Octave com diretório do navegador de arquivos</translation>
+        <translation>Sincronizar directório do Octave com directório do navegador de ficheiros</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Alternating row colors</source>
         <translation>Cores de linhas alternantes</translation>
     </message>
     <message>
         <location line="+21"/>
@@ -1608,17 +1614,17 @@ para:</translation>
     <message>
         <location line="+54"/>
         <source>Port:</source>
         <translation>Porta:</translation>
     </message>
     <message>
         <location line="-27"/>
         <source>Username:</source>
-        <translation>Nome de usuário:</translation>
+        <translation>Nome do utilizador:</translation>
     </message>
     <message>
         <location line="+37"/>
         <source>Password:</source>
         <translation>Senha:</translation>
     </message>
     <message>
         <location filename="../src/settings-dialog.cc" line="+70"/>
@@ -1681,17 +1687,17 @@ para:</translation>
     <message>
         <location filename="../src/qtinfo/webinfo.cc" line="+79"/>
         <source>Type here and press &apos;Return&apos; to search</source>
         <translation>Digite aqui e pressione &apos;Enter&apos; para buscar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Global search</source>
-        <translation>Busca global</translation>
+        <translation>Procura global</translation>
     </message>
 </context>
 <context>
     <name>welcome_wizard</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+40"/>
         <source>Welcome to GNU Octave</source>
         <translation>Bem-vindo ao GNU Octave</translation>
@@ -1699,22 +1705,22 @@ para:</translation>
     <message>
         <location line="+14"/>
         <source>Welcome to Octave!</source>
         <translation>Bem-vindo ao Octave!</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>You seem to be using the Octave graphical interface for the first  time on this computer.  Click &apos;Finish&apos; to write a configuration file  and launch Octave GUI.</source>
-        <translation>Parece que você está usando a interface gráfica do Octave pela primeira vez neste computador. Clique &apos;Finalizar&apos; para gravar um arquivo de configuração e iniciar a interface do Octave.</translation>
+        <translation>Parece que você está a usar a interface gráfica do Octave pela primeira vez neste computador. Clique &apos;Finalizar&apos; para escrever um ficheiro de configuração e iniciar o Octave.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The configuration file is stored in %1. If that file exists, you will not see this dialog when Octave starts again.</source>
-        <translation>O arquivo de configuração é armazenado em %1. Se o arquivo existir, você não verá esta caixa de diálogo quando o Octave reiniciar.</translation>
+        <translation>O ficheiro de configuração é armazenado em %1. Se o ficheiro existir, este diálogo não volta aparecer quando o Octave reiniciar.</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
@@ -1797,17 +1803,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Rename</source>
         <translation>Renomear</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Only top-level symbols may be renamed.</source>
-        <translation>Somente símbolos do nível mais alto podem ser renomeados.</translation>
+        <translation>Apenas símbolos do nível maisl alto podem ser renomeados.</translation>
     </message>
     <message>
         <location line="+125"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
         <translation>Visualizar as variáveis no ambiente de trabalho ativo.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
diff --git a/libgui/languages/translators b/libgui/languages/translators
--- a/libgui/languages/translators
+++ b/libgui/languages/translators
@@ -3,11 +3,12 @@
 # The generic file that can be used as a template for new
 # translation files is en_US.ts
 be_BY Mihas Varantsou <meequz@gmail.com>
 de_DE Jacob Dawid <jacob.dawid@googlemail.com>
 en_US Jacob Dawid <jacob.dawid@googlemail.com>
 es_ES Valentin Ortega-Clavero <arcanos3030@gmail.com>
 fr_FR David Bateman <dbateman@free.fr>, Catalin Codreanu <codreanu.catalin@gmail.com>
 pt_BR Júlio Hoffimann Mendes <julio.hoffimann@gmail.com>
+pt_PT Carnë Draug <carandraug@octave.org>
 ru_RU Andriy Shinkarchuck <adriano32.gnu@gmail.com>
 uk_UA Andriy Shinkarchuck <adriano32.gnu@gmail.com>
 nl_NL Sander van Rijn <svr003@gmail.com>
